src/main/java/com/mystays/authorizationserver/model/AuthenticationResponse.java

```diff
@@ -0,0 +1,17 @@
+package com.mystays.authorizationserver.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.*;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class AuthenticationResponse {
```

Comment:
May be we can have a separate AuthenticationResponse class. and this class can be changed to ErrorResponse to handle all the error responses in common.